components/cards/ProfileCard.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import {
+    Card,
+    CardBody,
+    Stack,
+    Text,
+    Box, Image,
+} from "@chakra-ui/react";
+import SearchIcon from "../icons/editIcon";
+
+
```

Comment:
Create a interface for ProfileCard with all properties as optional:
- image
- username
- location
- bio

then pass these.